packages/ui/src/camel-utils/camel-random-id.test.ts

```diff
@@ -13,6 +13,10 @@ describe('camel-random-id', () => {
   });
 
   it('should return a random number using Date.now() if crypto module is not available', () => {
+    Object.defineProperty(global, 'msCrypto', {
+      value: undefined,
+      writable: true,
+    });
```

Comment:
I'm not sure why this is global property is defined - is it because of the above defined mockImplementation?
```
jest
  .spyOn(global, 'crypto', 'get')
  .mockImplementation(....
```